.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+
+python:
+  - 3.4
```

Comment:
A couple things:
- We can remove 3.4 since it's already EOL: https://www.python.org/downloads/release/python-343/
- Let's also remove pypy since it's Python 2.7.
- Let's change "pypy3.5" to "pypy3". "pypy3" points to the latest pypy3 which is 3.6.

Reply:
Done